Description: Change from Fedora/RH/Centos default of Apache being run as user apache, to the Debian standard of www-data.
Author: Dave Walker (Daviey) <DaveWalker@ubuntu.com>
Forwarded: no
Last-Update: 2010-12-15

=== modified file 'Makefile'
--- old/Makefile	2010-06-17 14:38:33 +0000
+++ new/Makefile	2010-12-13 14:46:21 +0000
@@ -37,7 +37,7 @@
 
 install: build
 	python setup.py install -f
-	chown -R apache /usr/share/cobbler/web
+	chown -R www-data /usr/share/cobbler/web
 
 debinstall:
 	python setup.py install -f --root $(DESTDIR)
@@ -72,7 +72,7 @@
 	cp $(statepath)/dhcp.template /etc/cobbler/dhcp.template
 	cp $(statepath)/rsync.template /etc/cobbler/rsync.template
 	find /var/lib/cobbler/triggers | xargs chmod +x
-	chown -R apache /var/www/cobbler
+	chown -R www-data /var/www/cobbler
 	chmod -R +x /var/www/cobbler/web
 	chmod -R +x /var/www/cobbler/svc
 	rm -rf $(statepath)

=== modified file 'build_backup/Makefile'
--- old/build_backup/Makefile	2010-06-16 21:34:44 +0000
+++ new/build_backup/Makefile	2010-12-13 14:47:36 +0000
@@ -42,7 +42,7 @@
 
 install: build manpage
 	python setup.py install -f
-	chown -R apache /usr/share/cobbler/web
+	chown -R www-data /usr/share/cobbler/web
 
 debinstall: manpage
 	python setup.py install -f --root $(DESTDIR)
@@ -77,7 +77,7 @@
 	cp $(statepath)/dhcp.template /etc/cobbler/dhcp.template
 	cp $(statepath)/rsync.template /etc/cobbler/rsync.template
 	find /var/lib/cobbler/triggers | xargs chmod +x
-	chown -R apache /var/www/cobbler
+	chown -R www-data /var/www/cobbler
 	chmod -R +x /var/www/cobbler/web
 	chmod -R +x /var/www/cobbler/svc
 	rm -rf $(statepath)

=== modified file 'cobbler/action_reposync.py'
--- old/cobbler/action_reposync.py	2010-08-03 20:19:41 +0000
+++ new/cobbler/action_reposync.py	2010-12-13 14:46:48 +0000
@@ -563,7 +563,7 @@
         a safeguard.
         """
         # all_path = os.path.join(repo_path, "*")
-        cmd1 = "chown -R root:apache %s" % repo_path
+        cmd1 = "chown -R root:www-data %s" % repo_path
         utils.subprocess_call(self.logger, cmd1)
 
         cmd2 = "chmod -R 755 %s" % repo_path

=== modified file 'cobbler/cobblerd.py'
--- old/cobbler/cobblerd.py	2009-10-25 15:28:10 +0000
+++ new/cobbler/cobblerd.py	2010-12-13 14:45:39 +0000
@@ -61,7 +61,7 @@
     fd.write(binascii.hexlify(data))
     fd.close()
     utils.os_system("chmod 700 /var/lib/cobbler/web.ss")
-    utils.os_system("chown apache /var/lib/cobbler/web.ss")
+    utils.os_system("chown www-data /var/lib/cobbler/web.ss")
     return 1
 
 def do_xmlrpc_tasks(bootapi, settings, xmlrpc_port):

